src/callstack.cr

```diff
@@ -31,6 +31,16 @@ struct CallStack
     dir
   end
 
+  # ANSI color and formatting escape codes
```

Comment:
Why not use `Colorize` for that?

Reply:
Wait, there's a builtin library for this in Crystal? Huh, did not know, will fix ASAP.

Reply:
Also @Sija, do you feel that the current colour scheme for errors is appropriate?

Reply:
A screenshot would help

Reply:
There is `Colorize` but it should not be used for this. Including it here would make it a mandatory dependency for each Crystal program. This should be avoided and as the current code shows, it's certainly easy enough to do without using a specialized component.

Reply:
@RX14 Here is the error log with this program (I placed it in `tmp/temp.cr`):

```crystal
def foo
  bar
end

def bar
  a = [1]
  a[2]
end

foo
```

![screen shot 2017-11-26 at 10 34 19 am](https://user-images.githubusercontent.com/14177193/33235830-80943a8c-d295-11e7-9ce8-a4912be526f0.png)